codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Headers.java

```diff
@@ -184,6 +186,14 @@ public static boolean isPseudoHeader(ByteString header) {
     Http2Headers clear();
 
     /**
+     * Returns an iterator over all HTTP/2 headers from this instance. The iteration order is as follows
+     *   1. All reserved headers come first (in no particular order).
+     *   2. Headers with multiple values appear in insertion order.
```

Comment:
"All non-reserved headers (in no particular order). Headers with multiple values will have their values appear in insertion order." ?